lib/index.html

```diff
@@ -0,0 +1,105 @@
+    .doc-main h1 { font-weight: 300; font-size: 3rem; line-height: 1; margin: 0 }
+    .doc-main h2 { font-weight: 300; font-size: 1.5em; line-height: 1.5; margin: .5em 0; max-width: 650px }
+    .doc-main h3 { font-weight: 600; font-size: 12px; margin: 2em 0 .5em; text-transform: uppercase }
+    .doc-main hr { border: 0; border-top: 1px solid #eee; margin: 4rem -7vw; width: calc(100% + 14vw) }
+    .doc-main ul { padding-left: 25px; list-style: url("data:image/svg+xml,%3Csvg width='25' height='10' viewBox='0 0 7 10' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M1 1l4 4-4 4' stroke='%23ccc' stroke-width='2' fill='none'/%3E%3C/svg%3E") }
+    .doc-main td code:only-child { display: block; padding: 1em; margin-right: 1em}
+    .doc-main th { font-weight: 600; font-size: 12px }
+    .doc-main table { width: 100% }
+
+    .doc-demo { background: #f0f5fe; padding: 1rem; margin: 1rem 0 }
+    .doc-demo summary { cursor: pointer; padding: 5px 1rem; font-weight: 600; font-size: 12px; border-top: 2px solid #fff }
+    .doc-demo details { margin: 1rem -1rem -1rem }
+    .doc-demo ul { list-style: none }
+
+    @keyframes fadeIn { from { max-height: 0 } }
+    @media(min-width:700px) {
+      .doc-menu { float: left; box-sizing: border-box; position: sticky; top: 0; padding: 70px 0; width: 270px; min-height: 100vh; border-right: 1px solid #eee }
+    }
+  </style>
+</head>
+<body>
+  <script src="https://static.nrk.no/core-masthead/major/1/core-masthead.html.js"></script>
+  <div class="doc-menu">
+    <ul>
+      <li><a href="?docs.md">Core Icons</a></li>
+      <li><br><a href="https://github.com/nrkno/core-icons"><img src="https://cdn.jsdelivr.net/npm/simple-icons@latest/icons/github.svg"> View on Github</a></li>
+      <li><a href="https://github.com/nrkno/core-icons/releases"><img src="https://rawgit.com/nrkno/core-icons/master/lib/nrk-poll.svg"> View changelog</a></li>
+      <li><a href="https://github.com/nrkno/core-icons/issues/new?title=Icon%20Request:%20&labels=enhancement"><img src="https://rawgit.com/nrkno/core-icons/master/lib/nrk-plus.svg"> Request new icon</a></li>
+    </ul>
+  </div>
+  <main class="doc-main"></main>
+  <script src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
+  <script src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
+  <script src="https://unpkg.com/code-prettify/loader/run_prettify.js"></script>
+  <script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
+  <script src="https://unpkg.com/marked/marked.min.js"></script>
+  <script defer>
+    (function(){
+      var file = location.search || document.querySelector('.doc-menu a').search
+      var link = document.querySelector('.doc-menu a[href*="' + file + '"]')
+      var main = document.querySelector('.doc-main')
+      var ajax = new XMLHttpRequest()
+      var mark = new marked.Renderer()
+
+      mark.code = code
+      ajax.onload = function () {
```

Comment:
Blir delay nå på header/menyvalgene for `core-icons` pga vi laster inn docs.md etter hvert. Er det noe vi ønsker i dette tilfellet? Skal vi ha det fremfor en bygget dokumentasjonsside som har `docs.md` allerede inkludert?